sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -1768,8 +1337,11 @@ def create_page_blob(  # type: ignore
         :returns: Blob-updated property dict (Etag and last modified).
         :rtype: dict[str, Any]
         """
-        options = self._create_page_blob_options(
-            size,
+        options = _create_page_blob_options(
+            require_encryption=self.require_encryption,
+            key_encryption_key=self.key_encryption_key,
```

Comment:
For this one, let's take the one check for encryption outside of the helper function and not have to pass these.